README.md

```diff
@@ -11,16 +16,16 @@ $ npm install crocks -S
 This will pull down `crocks` into your project's `node_modules` folder and can be accessed by adding something like the following in the file that needs it:
 
 ```javascript
-// node require syntax
+// node require syntax - "es5"
 const crocks = require('crocks')
 
-// Javascript modules (if you are transpiling or using rollup.js)
+// Javascript modules (if you are transpiling or using rollup.js) - "es6"
 import crocks from 'crocks'
 ```
 
-There is no compilation going on here, so as a result, you will need to bundle or run with node `4.x` or higher. This lib *should* work, with no additional compilation in all current browsers (Edge, Safari, Chrome, Firefox), if it does not, please file an issue as I really, really want it to. :smile_cat:.
+There is no compilation going on here, so as a result, you will need to bundle or run with node `4.x` or higher.
 
-Another thing to note is, if you are transpiling, then destructuring in your `import` statement is not going to work as you are thinking (maybe if you are using `babel`, but this will be broken once modules are available in node, so be careful). Basically you should not do this, as `crocks` will not be set up for it until modules are available in node:
+**Precaution**: if you are de-structuring inline your `import` statement, you will need to use `babel` as this will not be supported once modules are available in node. Basically you should not do this, as `crocks` will not be set up for it until modules are available in node:
```

Comment:
It's again for readability; it's not very common to detail how `destructuring` works in a library anyways, so it could probably be pulled out altogether